client/my-sites/patterns/components/readymade-templates/index.tsx

```diff
@@ -61,12 +65,16 @@ export const ReadymadeTemplates = () => {
 		return;
 	}
 
+	const props = forwardRef ? { forwardRef } : {};
+
 	return (
 		<PatternsSection
+			{ ...props }
```

Comment:
Nitpick (non-blocking): Considering the number of allowed props we have right now, I think this is more legible if we define them here explicitely.


```suggestion
			forwardRef={ forwardRef }
```

Reply:
Same here. If we don't pass the forwardRef, we shouldn't add a null one.

Reply:
Is there any difference between passing a `null` forwaredRef and not passing it?